client/src/services/participant.js

```diff
@@ -44,12 +44,18 @@ export const fetchParticipant = async ({ id }) => {
     const { participant } = await resp.json();
     const cohort = await fetchParticipantCohort({ id });
     const postHireStatus = await fetchParticipantPostHireStatus({ id });
+    const status = participant.latestStatuses.length
+      ? participant.latestStatuses[0].status
+      : 'available';
+
     return {
       ...participant,
       cohort,
       cohortName: getCohortPsiName(cohort),
       postHireStatus,
       postHireStatusLabel: getPostHireStatusLabel(postHireStatus),
+      status,
+      siteName: participant.latestStatuses?.[0]?.siteName || null,
```

Comment:
Instead of latestStatuses[0], I might suggest a .find here (where status = 'hired'). I don't think we can always guarantee that hired is the 0th status.

Reply:
The status is being populated from `latestStatuses[0]` line 47
```const status = participant.latestStatuses.length
      ? participant.latestStatuses[0].status
      : 'available';```
      
I wanted to display the site, that is associated with the latest status.

Reply:
I guess I'm guarding against the possibility that, for example, there's two current statuses:
offer_made and hired (a valid scenario), and latestStatuses[0] is the offer made one
or
hired and reject_ack and archived (another valid scenario), and latestStatuses[0] is the reject one

Reply:
Should the UI display all valid statuses? (and associated sites next to the status)?

Reply:
No, we're just doing hired site. My worry is if [0] doesn't capture the hired status, it might be missed!

Reply:
Updated as we discussed on the call!